.github/workflows/publish-corfu.yml

```diff
@@ -1,15 +1,20 @@
-name: CorfDb Artifacts
-
-on:
-  push:
-    branches:
-      - master
+name: CorfuDB Artifacts
+#
+#on:
+#  push:
+#    branches:
+#      - master
+on: pull_request
 
 jobs:
   publish_corfu_db_artifacts:
 
     runs-on: ubuntu-latest
-    if: github.event_name == 'push'
+#    if: github.event_name == 'push'
+    env:
+      JFROG_PASSWORD: ${{ secrets.jfrog_oss_password }}
+      DOCKER_USERNAME: ${{ secrets.docker_username }}
+      DOCKER_PASSWORD: ${{ secrets.docker_password }}
```

Comment:
Do we really need this env change? Can we directly fetch from secrets in all the pom.xml files?

Reply:
As we discussed yesterday, that's the reason for this PR. Secrets cannot be directly accessed in other xml files.

Reply:
Ohh yeah, thank you!